### DISCUSSION TITLE
Is prismjs.com maintained by community? If so where do i report security issues?

### DISCUSSION THREAD

**Discussion by d34dman** [PrismJS/prism#2904] [Q&A] [answered]
the examples on https://prismjs.com/ for cdn has malicious script. The links for script and stylesheets are compromised.
<img width="1316" alt="Prism 2021-05-15 03-29-30" src="https://user-images.githubusercontent.com/1006481/118344307-d49ac280-b52d-11eb-874c-73d0b18d077c.png">

**Comment by mAAdhaTTah** [comment, marked as answer]
Thank you for letting us know! Fixed in [this PR](https://github.com/PrismJS/prism/pull/2905).

**Reply by RunDevelopment** [reply]
Holy that's dangerous! The PR has been merged and the compromised links are now gone. Thank you for letting us know @d34dman and thank you for fixing @mAAdhaTTah!

**Reply by d34dman** [reply]
Good job :) really appreciate the quick workaround :)

**Reply by d34dman** [reply]
I created a follow up PR #2906  to improve the user experience. It's not critical and totally optional.
